### ISSUE TITLE
0000105 folder (in Training Set1) dosen't exist.

### ISSUE THREAD

**Issue by yh97yhyh** [SamYuen101234/Masked_Face_Recognition#20]
Hi,
0000105 folder (in Training Set1) dosen't exist.
Could you upload that data?

<img width="550" alt="image" src="https://user-images.githubusercontent.com/47898473/158518129-9d2e96ef-81d1-4091-ae61-13582dcabe92.png">

**Comment by SamYuen101234** [comment]
Dear YoungHyun,

I have uploaded all the training data already. If someone is missed, you
can just delete the row in the training csv. These missing folders were
deleted because of mislabeling or wrong masking. The rest of the data is
enough for you to train a masked facial recognition model.

Regards,
Sam

YoungHyun Kim ***@***.***> 於 2022年3月16日週三 下午12:43寫道：

**Comment by yh97yhyh** [comment]
Thanks!! I'll try

**Comment by SamYuen101234** [comment]
closed
